src/ec/packages/ec-component-social-media-follow/docs/social-media-follow.md

```diff
@@ -0,0 +1,60 @@
+# ECL Twig - EC Social Media Follow component
+
+npm package: `@ecl-twig/ec-component-social-media-follow`
+
+```shell
+npm install --save @ecl-twig/ec-component-social-media-follow
+```
+
+## Social Media Follow
+
+### Parameters
+
+- "description" (string) (default: '')
+- "variant" (string) (default: '') - Can be 'vertical'
+- "links" (array) (default: []) - List of links to external social media. Each link consists of the following:
+  - "path" (string) - Namely the same attribute
+  - "label" (string) - Human-readable name of the link
+  - "icon_position" (string) - Whether the icon is expected to be "before" or "after"
+  - "variant" (string) - Link variant, usually "standalone" is default.
+  - "icon" (array) - List of icons used for normal and hover states. Each icon consists of the following:
+    - "name" (string) - Icon name
+    - "size" (string) - Size such as "xl"
+    - "path" (string) - Path in terms of an SVG icon
+    - "extra_classes" (string) - Class to toggle between normal and hover effects
+- "extra_classes" (optional) (string) (default: '') - Extra classes (space separated) for the icon
+- "extra_attributes" (optional) (array) (default: []) - Extra attributes for icon
+  - "name" (string) Attribute name, eg. 'data-test'
+  - "value" (string) Attribute value, eg: 'data-test-1'
+
+### Example:
+
+<!-- prettier-ignore -->
+```twig
+{% include 'path/to/social-media-follow.html.twig' with {
```

Comment:
found a little thing left: please add an extra space at the end of each line in the example. The goal is to force a new line in md. If you have a look at the code in the "note" tab on the website, you will see that it is rendered on a single line for now